### PR TITLE
added news about Jackalope development

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: src/Cmf/MainBundle/Resources/data/posts/2015-05-02.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  All users should see performance and reliability improvements. This is especially true for users
+  of the Doctrine DBAL transport where performance should have made a significant jump forward.
+  Note that in order to use Doctrine DBAL transport in the new version you will need to update
+  your database schema. Aside from this we also improved the maintainability and code-reuse
+  between the transports. Of course we also always expand the test suite when we do any work
+  on the code base. As we did fix several edge cases, it is recommend to review the CHANGELOG.md
+  and UPGRADE.md when encountering issues after the update.
+
+  For the future we work has also begun on [Jackalope2](https://github.com/jackalope2) where we
```
Comment (dbu) [inline src/Cmf/MainBundle/Resources/data/posts/2015-05-02.yml]: At the hackday we also discussed about a more radical refactoring of Jackalope to create a version 2 of Jackalope that will be more modular and have a cleaner architecture, allowing better code reuse and performance. Adding new storage or search solutions should become really simple.

We started drafting the concept in the [Jackalope2 wiki](https://github.com/Jackalope2/jackalope/wiki). If you want to get involved, please get in contact with us on [the mailinglist](https://groups.google.com/forum/#!forum/jackalope-dev) or in the Jackalope2 issue tracker on github.
